android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -829,6 +845,54 @@ private void finishPlayerInitialization() {
         }
     }
 
+    private static ExoMediaDrm createFrameworkMediaDrm(UUID uuid) {
+        try {
+            return VirtuosoFrameworkMediaDrm.newInstance(uuid);
+        } catch (UnsupportedDrmException e) {
+            e.printStackTrace();
```

Comment:
Similar comment here, throwing the exception is clearer, returning null will just cause a null pointer which is much harder to follow.